sbpy/spectroscopy/sources.py

```diff
@@ -498,3 +521,19 @@ def __repr__(self):
     @property
     def T(self):
         return self._T
+
+
+class Reddening(BaseUnitlessSpectrum):
+    """Class to handle simple linear reddening.
+
+    Parameters
+    ----------
+    S : `~SpectralGradient`
+        The spectral gradient to redden.
+    """
+    def __init__(self, S):
+        wv = [1, 2] * S.wave0
+        df = (S.wave0 * S).to('').value
+        super().__init__(
+            synphot.Empirical1D, points=wv, lookup_table=[1, 1+df],
```

Comment:
I don't see a Linear1D model in synphot, so I think Empirical1D is the right choice.  The only problem might be extrapolation beyond the two wavelength points, for which synphot has several options.  So what can we do to avoid or mitigate that.  We can certainly provide a data point for `wavelength = 0`.  Maybe also a very large value, e.g., 1 cm or `100 * wave0`?

Reply:
By setting `fill_value=None` to `Empirical1D`, the model will extrapolate outside of the supplied wavelength range.  When only two points are supplied, the extrapolation is linear.  This way, we can avoid having to provide 0 and a very large value for wavelength.

Reply:
You're right, it does work.  I was expecting a synphot DisjointError when trying to observe the result with, e.g., a filter that does not overlap the wavelength range specified here, but that doesn't seem to be an issue.